division.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
+    return x / y
```

Comment:
This doesn't look quite right

```suggestion
    return x / z
```